src/msix/pack/BundleValidationHelper.cpp

```diff
@@ -0,0 +1,248 @@
+#include "BundleValidationHelper.hpp"
+
+namespace MSIX {
+
+    const std::string Win10DeviceFamilyNameStartsWith = "Windows.";
+    const std::string OSVersionRS5 = "10.0.17763.0";
+
+    BundleValidationHelper::BundleValidationHelper() 
+    {}
+
+    void BundleValidationHelper::ValidateTargetDeviceFamiliesFromManifestPackageId(IAppxManifestPackageIdInternal* packageId, IAppxManifestTargetDeviceFamiliesEnumerator* tdfs, std::string fileName)
+    {
+        std::string arch = packageId->GetArchitecture();
+        if (arch == "neutral")

```

Comment:
>"neutral" [](start = 20, length = 9)

make this a const std::string as Win10DeviceFamilyNameStartsWith and use it i this method

Reply:
Updated.